src/clientsettingsdlg.cpp

```diff
@@ -244,6 +244,8 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     butDriverSetup->setAccessibleName ( tr ( "ASIO Device Settings push button" ) );
```

Comment:
Are the changes to `src/clientsettingsdlg.cpp` and `src/clientsettingsdlgbase.ui` relevant to this change?

Reply:
By "this change" you mean the PR as a whole? Yes, since we aren't quitting Jamulus when the sound card fails, we need to provide the user with more actions to deal with that state.